src/Diff/Internal/AliasGroupListPatcher.php

```diff
@@ -37,7 +36,7 @@ public function patchAliasGroupList( AliasGroupList $groups, Diff $patch ) {
 	}
 
 	/**
-	 * @see MapPatcher
+	 * @see Diff\Patcher\MapPatcher
```

Comment:
I am confused by this evil tool. So my Phpstorm recognizes MapPatcher in this @see link when not using FQN. I thought it only happens with "use" import but apparently it is smart enough even when "use" is not there.
It seems like a tool-specific thing, I don't care then.

Per this part of your comment:
> What you have seen are @return, @param and @var tags. This here is a @see. 

So what you're saying is there is a rule that imported classes are considered while resolving @return, @param and @var comment but not in case of a @see? Fine if there is something like this somewhere but it seems like a good reason to be confused about :)